### PR TITLE
[ART-1188] publish rpms to mirror

### PR DESCRIPTION
**Pull request by Ximinhan**
Automate publishing rpms to mirror, every time a new pre-release accepted update latest RHEL 7 worker RPMs on mirror.

### CODE REVIEW

**Comment by Ximinhan** [comment]
Here is the test job https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/hack/job/ximhan-art-1188/job/build%252Fpublish-rpms/5/console , I run the test job against 4.1 there are some error in output shows the mirror link is unreachable but the whole job return success.

**Review by joepvd — APPROVED** [review]
Left some suggestions. Those should not block a merge, so approve :+1:

File: jobs/build/publish-rpms/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: ''
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    string(
```
[the remark this answers is not in the document]
Comment (Ximinhan) [reply jobs/build/publish-rpms/Jenkinsfile:25]: I think this is job used for next version of release, in this case is 4.7 maybe next time is 4.8.So it's hard to determine the scope of list.

**Review by joepvd — APPROVED** [review]
/lgtm

**Pull request merged**